package.json

```diff
@@ -1,34 +1,39 @@
 {
```

Comment:
If you do a `npm install --save-dev X` or `npm install --save Y` it will autoformat the `package.json` to be 2 spaces. I'm guessing something like that happened. I think it's easier to read with 4. But it may be too much effort to try to enforce with so many people working on the project.